### DISCUSSION TITLE
安装完为什么显示不了啊？

### DISCUSSION THREAD

**Discussion by amlhdsan** [rime/home#1707] [Q&A] [answered]
我不管使用dmg安装包还是brew命令行安装后，重启什么的重新登录什么的都不行，都显示不出来鼠须管的图标，永远都是Apple的可恶的“简体拼音”，怎么办啊？急死了，在线等。

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
去設置-鍵盤-輸入法添加一下

**Comment by amlhdsan** [comment]
可以用簡體中文嗎？只有繁體中文嗎？
